stl/inc/tuple

```diff
@@ -186,7 +186,7 @@ template <class... _Types, class _Other, size_t... _Indices>
 inline constexpr bool _Can_construct_values_from_tuple_like_v<tuple<_Types...>, _Other, index_sequence<_Indices...>> =
     conjunction_v<is_constructible<_Types, decltype(_STD get<_Indices>(_STD declval<_Other>()))>...>;
 
-#ifdef __clang__ // TRANSITION, LLVM-59827
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, LLVM-59827 (Clang), VSO-1900279 (EDG)

```

Comment:
We haven't annotated which bug corresponds to which compiler in the past, and I'm not sure how useful it is. Certainly it's clear that LLVM-XXXXX is always a Clang issue, and by process of elimination even folks without access to the Microsoft-internal VSO bug database can determine the other is EDG-specific. I'm mildly inclined to remove `(Clang)` and `(EDG)` from this and similar comments as noise.

Reply:
> I'm mildly inclined to remove `(Clang)` and `(EDG)` from this and similar comments as noise.

Done.

We do occasionally mention `(EDG)` in comments, when it's not obvious that the bug is EDG-specific (e.g. https://github.com/microsoft/STL/blob/b84bf09ab6f9267992f899956cea46e7824fe0c3/stl/inc/format#L1013). But I agree that it's not needed here.